.github/workflows/ci.yml

```diff
@@ -194,14 +192,9 @@ jobs:
       - name: cache rust
         uses: Swatinem/rust-cache@v1
 
-      - run: pip install nox requests
-
-      - name: setup pyodide
-        run: nox -f tests/noxfile.py -s setup-pyodide
-
       - uses: mymindstorm/setup-emsdk@v11
         with:
-          version: ${{ env.EMSCRIPTEN_VERSION }}
+          version: '3.1.14'
```

Comment:
The previous `setup-pyodide` reads required emscripten version from `repodata.json` from pyodidie so that when upgrading pyodide you don't need to figure out what emscripten version it requires.

This change loses that benefit but I guess it's acceptable since CI will fail if you don't set the right version.

Reply:
I was even hoping to go the other way and also read the version of Python from `repodata.json`. After the Pyodide v0.21.0 release hopefully we can set up a github action to automate all this setup.

Reply:
makes sense, we could probably have documented that with a comment, but it was note clear at all.

I think sometimes repetition is a fair price to pay for (relative) simplicity.